angular/src/polyfills.ts

```diff
@@ -22,32 +22,32 @@ import 'whatwg-fetch';
 /**
  * IE9, IE10 and IE11 requires all of the following polyfills.
  */
-import 'core-js/es6/array';
-import 'core-js/es6/date';
-import 'core-js/es6/function';
-import 'core-js/es6/map';
-import 'core-js/es6/math';
-import 'core-js/es6/number';
-import 'core-js/es6/object';
-import 'core-js/es6/parse-float';
-import 'core-js/es6/parse-int';
-import 'core-js/es6/regexp';
-import 'core-js/es6/set';
-import 'core-js/es6/string';
-import 'core-js/es6/symbol';
-import 'core-js/es6/weak-map';
+import 'core-js/es/array';
+import 'core-js/es/date';
+import 'core-js/es/function';
+import 'core-js/es/map';
+import 'core-js/es/math';
+import 'core-js/es/number';
+import 'core-js/es/object';
+import 'core-js/es/parse-float';
+import 'core-js/es/parse-int';
+import 'core-js/es/regexp';
+import 'core-js/es/set';
+import 'core-js/es/string';
+import 'core-js/es/symbol';
+import 'core-js/es/weak-map';
 
 /** IE10 and IE11 requires the following for NgClass support on SVG elements */
 import 'classlist.js'; // Run `npm install --save classlist.js`.
 
 /** IE10 and IE11 requires the following for the Reflect API. */
-import 'core-js/es6/reflect';
+import 'core-js/es/reflect';
 
 /**
  * Evergreen browsers require these.
  */
 // Used for reflect-metadata in JIT. If you use AOT (and only Angular decorators), you can remove.
-import 'core-js/es7/reflect';
+import 'core-js/es/reflect';
```

Comment:
Do we need this twice since it's been updated to be from one package?

Reply:
Hmm, good thinking. We don't need it twice. I'll remove it.